Source/Core/Core/DSP/Jit/x64/DSPJitBranch.cpp

```diff
@@ -273,9 +273,9 @@ void DSPEmitter::rti(const UDSPInstruction opc)
 // HALT
 // 0000 0000 0020 0001
 // Stops execution of DSP code. Sets bit DSP_CR_HALT in register DREG_CR.
-void DSPEmitter::halt(const UDSPInstruction opc)
+void DSPEmitter::halt(const UDSPInstruction)
```

Comment:
Since the argument isn't used at all, is there a point in keeping the const qualifier?

Reply:
I kept it for consistency with the rest of the instructions

Reply:
makes sense